libs/designsystem/button/src/button.component.ts

```diff
@@ -20,6 +20,8 @@ export enum ButtonSize {
   LG = 'lg',
 }
 
+export type AttentionLevel = '1' | '2' | '3' | '4';
+
```

Comment:
Should this be exported so the implementer can see it?

Reply:
Yes, I forgot to expose it